CMakeLists.txt

```diff
@@ -462,6 +462,9 @@ OCV_OPTION(WITH_ANDROID_MEDIANDK "Use Android Media NDK for Video I/O (Android)"
 OCV_OPTION(WITH_ANDROID_NATIVE_CAMERA "Use Android NDK for Camera I/O (Android)" (ANDROID_NATIVE_API_LEVEL GREATER 23)
   VISIBLE_IF ANDROID
   VERIFY HAVE_ANDROID_NATIVE_CAMERA)
+OCV_OPTION(WITH_TENGINE "Include Arm Inference Tengine support" OFF
+  VISIBLE_IF (ARM OR AARCH64) AND (UNIX OR ANDROID) AND NOT IOS AND NOT VISIONOS
+  VERIFY HAVE_TENGINE)
```

Comment:
TEngine support was dropped in some previous PRs as not supported by the original author. Please rebase on current 4.x.

Reply:
Rebased and force pushed